<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HR0UEC8" failures="1" tests="2" name="Command line test" time="0.097" errors="0" timestamp="2022-09-22T14:57:17 SGT">
  <testcase classname="TestNGTest" name="test" time="0.007"/>
  <testcase classname="TestNGTest" name="test1" time="0.031">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1]
at TestNGTest.test1(TestNGTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- Command line test -->
